pandas/tests/extension/test_integer.py

```diff
@@ -163,6 +168,7 @@ def test_error(self, data, all_arithmetic_operators):
 
 class TestComparisonOps(base.BaseComparisonOpsTests):
     def check_opname(self, s, op_name, other, exc=None):
+        pytest.skip(msg="TODO: NA comparisions")
```

Comment:
Should I implement NA comparisons here, or leave that as a followup? Right now we have the `np.nan` always-false behavior.

```python
In [6]: pd.Series([1, 2, pd.NA], dtype="Int64") > 1
Out[6]:
0    False
1     True
2    False
dtype: bool
```

Reply:
I think you started adding it here? (which is fine I think), but so can remove this skip again. 
Although you might want to skip those tests fully given the deviating new behaviour, it might be more annoying than worth it to try to get those tests working. 
I see that in the BooleanArray PR I skip the two tests explicitly: https://github.com/pandas-dev/pandas/pull/29961/files#diff-c2269e718ef70677ad49e7aea9beda42R163-R170 (so to keep the third test which is still relevant, or if there are added others in the future)